css/civicrm.css

```diff
@@ -3173,6 +3173,18 @@ div.m ul#civicrm-menu,
 .crm-container .even.crm-row-selected {
   background-color: #fffdb2;
 }
+.crm-container table tr.merge-row-equal {
+  background-color: #EFFFE7;
+  border-bottom: 1px solid #ccc;
+}
+.crm-container table tr.merge-row-diff {
+  background-color: #FFECEC;
+  border-bottom: 1px solid #ccc;
+}
+.crm-container table tr.merge-row-equal.crm-row-selected,
+.crm-container table tr.merge-row-diff.crm-row-selected {
+  background-color: #fffdb2;
+}
```

Comment:
I'm trying to avoid putting css that pertains to a single screen into our global civicrm.css file.
We could put this in admin.css, or better yet, start a new css file just for deduping.